acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerProcessRunner.java

```diff
@@ -107,6 +82,17 @@ protected void startExecutor(final List<String> params) {
     }
   }
 
+  private void addTrustStoreOptions(final StringJoiner javaOpts) {
+    if (getSignerConfig().getOverriddenCaTrustStore().isPresent()) {
+      final TlsCertificateDefinition caTrustStore =
+          getSignerConfig().getOverriddenCaTrustStore().get();
+      final Path overriddenCaTrustStorePath = createJksCertFile(caTrustStore);
+      javaOpts.add(
+          "-Djavax.net.ssl.trustStore=" + overriddenCaTrustStorePath.toAbsolutePath().toString());
+      javaOpts.add("-Djavax.net.ssl.trustStorePassword=" + caTrustStore.getPassword());
```

Comment:
can we do something nicer - like return the javaopts, arther than modifying the passed in one?

Reply:
done